src/sagemaker/session.py

```diff
@@ -4843,6 +4844,7 @@ def _create_inference_recommendations_job_request(
             framework (str): The machine learning framework of the Image URI.
             sample_payload_url (str): The S3 path where the sample payload is stored.
             supported_content_types (List[str]): The supported MIME types for the input data.
+            model_name (str): Name of the Amazon SageMaker ``Model`` to be used.
```

Comment:
because model creation etc. will happen underneath, lets expand on this more. The API Docs will be generated from this doc string